sdk/identity/Azure.Identity/src/IdentityClient.cs

```diff
@@ -132,7 +132,7 @@ private Request CreateClientSecretAuthRequest(string tenantId, string clientId,
         {
             Request request = _pipeline.CreateRequest();
 
-            request.Method = HttpPipelineMethod.Get;
+            request.Method = HttpPipelineMethod.Post;
```

Comment:
@schaabs I changed from GET to POST here, .NET Framework doesn't allow get requests with body and spec seems to allow POSTs to this endpoint.